content/pages/healthcare/apply/application.md

```diff
@@ -9,7 +9,7 @@ maintenance_line2: In the meantime, you can still call 1-877-222-VETS(8387) and
 ---
 <div id="main">
   <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
+    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
```

Comment:
This is the standard markup for breadcrumbs now; without it there's some extra left margin.